.github/workflows/ci.yml

```diff
@@ -42,8 +42,10 @@ jobs:
       run: scripts/xcode_select_current_version.sh
     - name: Install xcbeautify
       run: brew install xcbeautify
-    - name: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }} | xcbeautify
-      run: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }} | xcbeautify
+    - name: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }} | xcbeautify --renderer github-actions
+      run: |
+        set -o pipefail
```

Comment:
If you do `set -eox`, then bash will print out the executable part of the script in it's run which is kinda useful for debugging :P

Reply:
oooh